test/support_stdlib/testsuite_common_types.h

```diff
@@ -0,0 +1,98 @@
+// -*- C++ -*-
+// typelist for the C++ library testsuite.
+//
+// Copyright (C) 2005-2019 Free Software Foundation, Inc.
+//
+// This file is part of the GNU ISO C++ Library.  This library is free
+// software; you can redistribute it and/or modify it under the
+// terms of the GNU General Public License as published by the
+// Free Software Foundation; either version 3, or (at your option)
+// any later version.
```

Comment:
Is this the right license? I guess it should be `Apache License v2.0 with LLVM Exceptions` as in the other tests. 

Also, there are some files without license e.g. `test/support_stdlib/testsuite_iterators.h`. Could you specify a license there?

Reply:
@SergeyKopienko, what's your answers here?

Reply:
My answer: it's license from the source test file so I prefer to keep it.

Reply:
File deleted as unused.